src/go/cmd/update-collection-v3/migrations/fluentd-logs-configs/fluentd.go

```diff
@@ -0,0 +1,175 @@
+package fluentdlogsconfigs
+
+func setSumologic(valuesV2 *ValuesV2) *SumologicV3 {
```

Comment:
The names of these functions are confusing to me. They're not setting anything, but rather, creating new configuration based on old one. I'd be fine with `get` or `create` or `migrate`.